bootstrap/argocd/components/sso/external-secret-argocd-sso.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: argocd-sso
+spec:
+  refreshInterval: 1h
```

Comment:
Can we force the sync? or does it monitor the source resources for changes?

Reply:
That's how often it checks the source for changes. I'm not sure how to force stuff without using an annotation. Which I would love to do but just don't know how yet.

Reply:
Found this:

`kubectl annotate es name-of-es force-sync=$(date +%s) --overwrite`